From: "Daniel W. S. Almeida" <dwlsalmeida@gmail.com>
Date: Sat, 18 Jul 2020 13:51:04 -0300
Subject: docs: bpf/index.rst: Add ringbuf.rst
Patch-mainline: v5.9-rc1
Git-commit: 5fe52b3865eacce7206ff7e28455c686e6bf8920
References: bsc#1177028

Fix the following warning

Documentation/bpf/ringbuf.rst: WARNING: document isn't
included in any toctree

By adding it to the index.

Signed-off-by: Daniel W. S. Almeida <dwlsalmeida@gmail.com>
Link: https://lore.kernel.org/r/20200718165107.625847-10-dwlsalmeida@gmail.com
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Acked-by: Gary Lin <glin@suse.com>
---
 Documentation/bpf/index.rst |    8 ++++++++
 1 file changed, 8 insertions(+)

--- a/Documentation/bpf/index.rst
+++ b/Documentation/bpf/index.rst
@@ -58,6 +58,14 @@ Testing and debugging BPF
    s390
 
 
+Other
+=====
+
+.. toctree::
+   :maxdepth: 1
+
+   ringbuf
+
 .. Links:
 .. _Documentation/networking/filter.txt: ../networking/filter.txt
 .. _man-pages: https://www.kernel.org/doc/man-pages/
